pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -104,37 +104,33 @@ func PrintDownStreamVersions(output io.Writer, minSupportedVersion string) error
 	return nil
 }
 
+// PrintUpStreamVersions prints the origin versions which satisfies the following conditions:
+// 	1. Major versions greater than or equal to the minimum supported and default version
+//	2. Pre-release versions greater than or equal to the minimum supported version which
+//	   do not have a major version
```

Comment:
@budhrg Will these comments do?

Reply:
@anjannath 
Change `Pre-release versions greater than or equal to the minimum supported version which do not have a major version` to 
```
Pre-release versions greater than default version
```

Reply:
@budhrg Thanks.. that's much simpler :)